hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -306,6 +306,10 @@ spec:
     autoUpdate:
       kubernetesVersion: ${value("maintenance.autoUpdate.kubernetesVersion", "true")}
   % if cloud != "local":
+  # Backup configuration for Shoot clusters is deprecated and no longer supported.
```

Comment:
Rather than commenting it here, isn't it better to remove this section all the way from examples?

Reply:
maybe hint that it is deprecated for now like the API? if users see that in only shoots that are reconciled with newer versions maybe that would be confusing?

Reply:
I basically aligned to what we did for the deprecated add-ons. Having an explanation in the examples gives the users a chance to understand that and why certain things in the API changed. Since we don't have any user documentation this is the only place we can actually put something like this.

Reply:
ok. 

That reminds me, since we don't have kubernetes like API reference documentation, does it make sense to put link to https://godoc.org/github.com/gardener/gardener/pkg/apis/garden on README.  I guess its better place for API documentation. Just a suggestion.

Reply:
@swapnilgm maybe in the future, but not now. We are introducing a new API group anyways and any effort in that direction seeps away.

Reply:
Cool. NP. 👍